Document third-party SBOM upload (#28891)
DataDog/documentation: content/en/security/code_security/software_composition_analysis/setup_static/_index.md

If you are using another source code management provider, configure SCA to run in your CI pipelines using the `datadog-ci` CLI tool and [upload the results][8] to Datadog.
You **must** run an analysis of your repository on the default branch before results can begin appearing on the **Code Security** page.

## Upload third-party SBOM to Datadog

> While Datadog preferred SBOM generator is [our own osv-scanner fork][10], it is possible to ingest

Reviewer: ```suggestion
While Datadog preferred SBOM generator is [our own osv-scanner fork][10], it is possible to ingest a
```

third-party SBOM.

Our tooling supports the following SBOM standards:
 - [CycloneDX 1.4][18]
 - [CycloneDX 1.5][19]
 - [CycloneDX 1.6][20]

When ingesting a third-party SBOM, ensure that the following constraints are met:

>  - the file checks the SBOM JSON schema

Reviewer: ```suggestion
 - The file checks the SBOM JSON schema
```

 - SBOM components have the type `library`
 - SBOM components have a valid `purl` attribute

Third-party SBOM files are uploaded to Datadog using the `datadog-ci` command. You can use the following

> command to upload your third-party SBOM (make sure the enviroment variables `DD_API_KEY`, `DD_APP_KEY` and `DD_SITE`

Reviewer: ```suggestion
command to upload your third-party SBOM. Ensure the environment variables `DD_API_KEY`, `DD_APP_KEY` and `DD_SITE`
```

> are respectively set to your API key, APP key and [Datadog site][12]).

Reviewer: ```suggestion
are set to your API key, APP key, and [Datadog site][12], respectively.
```
